### PR TITLE
feat(build): add table output

### PR DESCRIPTION
**Pull request by jbrockopp**
* Added a `table()` function to `go-vela/cli/action/build`
* Added tests for the `table()` function
* Added a `wideTable()` function to `go-vela/cli/action/build`
* Added tests for the `wideTable()` function
* Added `go-vela/cli/internal/output.Dump` as an output option
* Added `go-vela/cli/internal/output.Spew` as an output option

### CODE REVIEW

File: action/build/get.go [hunk 1]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
 
 	// handle the output based off the provided configuration
 	switch c.Output {
-	case "json":
-		// output the build in JSON format
+	case output.DriverDump:
```
Comment (jbrockopp) [inline action/build/get.go:29]: Each function now supports the `dump` driver from `go-vela/cli/internal/output`

File: action/build/get.go [hunk 2]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
+		if err != nil {
+			return err
+		}
+	case output.DriverJSON:
+		// output the builds in JSON format
 		err := output.JSON(builds)
 		if err != nil {
 			return err
 		}
+	case output.DriverSpew:
```
Comment (jbrockopp) [inline action/build/get.go:41]: Each function now supports the `spew` driver from `go-vela/cli/internal/output`

File: action/build/get.go [hunk 3]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
-		// TODO: create output.Wide function
-		//
-		// err := output.Wide(builds)
-		// if err != nil {
-		// 	return err
-		// }
-	case "yaml":
-		// output the build in YAML format
+		// output the builds in wide table format
+		err := wideTable(builds)
```
Comment (jbrockopp) [inline action/build/get.go:49]: Adding support for `wide` output option with a new `wideTable()` function

File: action/build/get.go [hunk 4]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
+	case output.DriverYAML:
+		// output the builds in YAML format
 		err := output.YAML(builds)
 		if err != nil {
 			return err
 		}
 	default:
-		// output the build in default format
-		err := output.Default(builds)
+		// output the builds in table format
```
Comment (jbrockopp) [inline action/build/get.go:60]: Updated the default output option with a new `table()` function

File: action/build/table.go [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+	"github.com/go-vela/types/library"
+
+	"github.com/dustin/go-humanize"
+	"github.com/gosuri/uitable"
+)
+
+// table is a helper function to output the
+// provided builds in a table format with
+// a specific set of fields displayed.
+func table(builds *[]library.Build) error {
```
Comment (jbrockopp) [inline action/build/table.go:24]: This is where we add the new `table()` function

File: action/build/table.go [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+		return err
+	}
+
+	return nil
+}
+
+// wideTable is a helper function to output the
+// provided builds in a wide table format with
+// a specific set of fields displayed.
+func wideTable(builds *[]library.Build) error {
```
Comment (jbrockopp) [inline action/build/table.go:58]: This is where we add the new `wideTable()` function

File: action/build/table_test.go [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+	b.SetMessage("First commit...")
+	b.SetCommit("48afb5bdc41ad69bf22588491333f7cf71135163")
+	b.SetSender("OctoKitty")
+	b.SetAuthor("OctoKitty")
+	b.SetEmail("OctoKitty@github.com")
+	b.SetLink("https://example.company.com/github/octocat/1")
+	b.SetBranch("master")
+	b.SetRef("refs/heads/master")
+	b.SetBaseRef("")
+	b.SetHost("example.company.com")
```
Comment (wass3r) [inline action/build/table_test.go:130]:
```suggestion
	b.SetHost("host.example.com")
```

File: action/build/table_test.go [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+	b.SetDeploy("")
+	b.SetClone("https://github.com/github/octocat.git")
+	b.SetSource("https://github.com/github/octocat/48afb5bdc41ad69bf22588491333f7cf71135163")
+	b.SetTitle("push received from https://github.com/github/octocat")
+	b.SetMessage("First commit...")
+	b.SetCommit("48afb5bdc41ad69bf22588491333f7cf71135163")
+	b.SetSender("OctoKitty")
+	b.SetAuthor("OctoKitty")
+	b.SetEmail("OctoKitty@github.com")
+	b.SetLink("https://example.company.com/github/octocat/1")
```
Comment (wass3r) [inline action/build/table_test.go:126]:
```suggestion
	b.SetLink("https://host.example.com/github/octocat/1")
```

**Pull request merged**